### PR TITLE
orange-book:0.1.0

### PR DESCRIPTION
**Pull request by flavio20002**
I am submitting
- [ X] a new package
- [ ] an update for a package

Description: A book template inspired by The Legrand Orange Book of Mathias Legrand and Vel

I have read and followed the submission guidelines and, in particular, I
- [X ] selected a name that isn't the most obvious or canonical name for what the package does
- [ X] added a `typst.toml` file with all required keys
- [ X] added a `README.md` with documentation for my package
- [ X] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [ X] tested my package locally on my system and it worked
- [ X] `exclude`d PDFs or README images, if any, but not the LICENSE

- [X ] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

**Comment by elegaanz** [comment]
Hi, as suggested by the automatic review, Typst packages generally use kebab-case names for functions and their arguments. I'll let you push another commit if you want to follow this convention. If you prefer to keep you package as is, just tell me, I can still merge it.

**Comment by flavio20002** [comment]
Hi, I'll fix it right now. Thanks!

**Comment by elegaanz** [comment]
Thanks!

File: packages/preview/orange-book/0.1.0/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# orange-book
+A book template inspired by The Legrand Orange Book of Mathias Legrand and Vel https://www.latextemplates.com/template/legrand-orange-book.
+
+## Usage
+You can use this template in the Typst web app by clicking "Start from template"
+on the dashboard and searching for `orange-book`.
+
+Alternatively, you can use the CLI to kick this project off using the command
+```
+typst init @preview/orange-book
```
Comment (talios) [inline packages/preview/orange-book/0.1.0/README.md:10]: Looks like this doesn't work - however adding the version does:

```
typst init @preview/orange-book:0.1.0
```

Comment (elegaanz) [reply packages/preview/orange-book/0.1.0/README.md:10]: I think the reason is that package index is only re-built once a day, so you may have to wait up to 24h before the compiler is aware that the 0.1.0 is the latest available.

**Pull request merged**